walkthroughs/1A-integrate-event-and-api-driven-apps/walkthrough.json

```diff
@@ -9,30 +9,652 @@
+                      "image": "quay.io/integreatly/nodejs-messaging-work-queue:3.0.1",
+                      "imagePullPolicy": "Always",
+                      "livenessProbe": {
+                        "httpGet": {
+                          "path": "/api/health/liveness",
+                          "port": 8080,
+                          "scheme": "HTTP"
+                        },
+                        "initialDelaySeconds": 60
+                      },
+                      "name": "nodejs-messaging-work-queue-frontend",
+                      "ports": [
+                        {
+                          "containerPort": 8080,
+                          "name": "http",
+                          "protocol": "TCP"
+                        }
+                      ],
+                      "readinessProbe": {
+                        "httpGet": {
+                          "path": "/api/health/readiness",
+                          "port": 8080,
+                          "scheme": "HTTP"
+                        }
+                      },
+                      "securityContext": {
+                        "privileged": false
+                      }
+                    }
+                  ]
+                }
+              },
+              "triggers": [
+                {
+                  "type": "ConfigChange",
+                  "imageChangeParams": {
+                    "automatic": true,
+                    "containerNames": [
+                      "nodejs-messaging-work-queue-frontend"
+                    ],
+                    "from": {
+                      "kind": "ImageStreamTag",
+                      "name": "nodejs-messaging-work-queue-frontend:latest"
+                    }
+                  }
+                }
+              ]
+            }
+          },
+          {
+            "apiVersion": "v1",
+            "kind": "Service",
+            "metadata": {
+              "labels": {
+                "app": "nodejs-messaging-work-queue-frontend"
+              },
+              "name": "nodejs-messaging-work-queue-frontend"
+            },
+            "spec": {
+              "ports": [
+                {
+                  "name": "http",
+                  "port": 8080
+                }
+              ],
+              "selector": {
+                "app": "nodejs-messaging-work-queue-frontend"
+              }
+            }
+          },
+          {
+            "apiVersion": "v1",
+            "kind": "Route",
+            "metadata": {
+              "labels": {
+                "app": "nodejs-messaging-work-queue-frontend"
+              },
+              "name": "frontend"
+            },
+            "spec": {
+              "port": {
+                "targetPort": 8080
+              },
+              "to": {
+                "kind": "Service"
+              }
+            }
           }
-        }
+        ],
+        "name": "nodejs-messaging-work-queue-frontend"
```

Comment:
This was added in the wrong place. It needs to be in `spec.to` of the route object. Example:
```
spec:
  host: frontend-admin-tutorial-f6e1.apps.omatskiv-5d7d.openshiftworkshop.com
  wildcardPolicy: None
  port:
    targetPort: 8080
  to:
    kind: Service
    name: nodejs-messaging-work-queue-frontend
```

Reply:
Thanks @matskiv busy looking into this now! Looks like there was an issue when I converted the yaml to json